open-metadata-implementation/access-services/glossary-view/glossary-view-api/src/main/java/org/odpi/openmetadata/accessservices/glossaryview/exception/GlossaryViewOmasException.java

```diff
@@ -7,14 +7,14 @@
 /**
  * Stores checked OMRS exceptions info until it is sent to the client. Made to be used only by {@code OMRSClient}
  */
-public class OMRSExceptionWrapper extends OMRSCheckedExceptionBase {
+public class GlossaryViewOmasException extends OMRSCheckedExceptionBase {
```

Comment:
OMAS Exceptions should extend the common services exceptions or OCF

Reply:
Changed extended class